tests/test_main.py

```diff
@@ -365,3 +365,17 @@ def test_not_render_rules_also_matched(z9324d_extractor, capsys):
     assert "act as TCP client" in std.out
     assert "connect TCP socket" in std.out
     assert "create TCP socket" in std.out
+
+
+# It tests main works with different backends. It doesn't test that the backend
+# is actually called.
+def test_backend_option(capsys):
+    if sys.version_info > (3, 0):
+        path = get_data_path_by_name("pma16-01")
+        assert capa.main.main([path, "-b", capa.main.BACKEND_VIV]) == 0
+        std = capsys.readouterr()
+        assert "check for PEB NtGlobalFlag flag (24 matches)" in std.out
```

Comment:
```suggestion
        assert "extractor            VivisectFeatureExtractor" in std.out
```

Reply:
Untested, maybe just use `assert "VivisectFeatureExtractor" in std.out`?

Reply:
should I remove the `check for PEB NtGlobalFlag flag (24 matches)` assert? This is the proof that capa is working, otherwise there could be a backend for which capa doesn't detect any rule. Or do you think that this is already tested in the features tests? 🤔 I selected this rule because we use this file (PMA 16 - 01) in the rule examples.

Reply:
Maybe we can use the `-j` option and assert that `len(matches) > 0`?